REQUIRE

```diff
@@ -1,3 +1,5 @@
 julia 0.6
 StatsBase
 Distributions
+PositiveFactorizations
+StatsModels 0.2.1
```

Comment:
Heh, technically the StatsModels tag should have been 0.3.0 since it introduces a new feature, but whatevs.

Reply:
We'll have to make a decision about this, but in my book major releases are only needed when breaking backward compatibility. Else, almost all of our releases need to be major ones...

Reply:
My understanding is that, per semantic versioning, patch releases (i.e. the z in x.y.z) can only contain fixes which do not introduce new behaviors or change existing behaviors. Minor releases (the y) should be used when introducing new features _or_ when making a backwards-incompatible change if x = 0. Major releases (the x) should be used for backwards-incompatible changes when x >= 1.

Reply:
Yeah, but before 1.0 we use the patch release as the minor release.

Reply:
Okay. I don't actually care that much anyway. :stuck_out_tongue: